addons/website_livechat/static/src/components/visitor_banner/visitor_banner.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="website_livechat.VisitorBanner" owl="1">
+        <div class="o_VisitorBanner">
+            <div class="o_VisitorBanner_sidebar">
+                <div class="o_VisitorBanner_avatarContainer">
+                    <img class="o_VisitorBanner_avatar rounded-circle" src="/mail/static/src/img/smiley/avatar.jpg" alt="Avatar"/>
+                    <t t-if="visitor.is_connected">
+                        <i class="o_VisitorBanner_onlineStatusIcon fa fa-circle" title="Online" role="img" aria-label="Visitor is online"/>
+                    </t>
+                </div>
+            </div>
+            <div class="o_VisitorBanner_content">
+                <t t-if="visitor.country">
+                    <img class="o_VisitorBanner_country o_country_flag" t-att-src="'base/static/img/country_flags/'+visitor.country.code+'.png'"/>
```

Comment:
visitor.country.image_url ?

Reply:
@JKE-be unfortunately this field is not returned by `website_livechat` methods where only `country_code` is given. As we are only refactoring JS for now (and as it's for a stable version) we try to avoid to change the format of python responses. By the way, it's a good remark for the future.

@alexkuhn do you agree with me or should we consider to make this field available by changing `channel_info` of `website_livechat` now or should we do that later ?

Reply:
Haven't checked PR yet, but for this part it's best to rely on `image_url`, as @JKE-be suggested.
Either adapt `channel_info` to provide more info about the `country`, or fetch `image_url` if we haven't it and we need it in template (could show spinner in the meantime).

Reply:
@seb-odoo told me to not change response format in stable version so if I understood it well, channel_info should not change. And it's kind of overkill to make a RPC call to get a field we can guess, no ?

The best it to edit channel_info but should be done later if I understood SEB well.

Reply:
@xdu-odoo Removing a key is always more risky than adding one (because another part of the code, a custo, etc, could rely on it, whereas adding one should usually do no harm). In this case I would be fine to add it if we think it's the best approach the solve the issue.

Reply:
Okay, didn't fully understand what you were saying then :)

Reply:
Mmh `country` seems to not have a `image_url` field ?

Reply:
Probably in 13.5, you can keep it like this in 13.4 and change it after.

Reply:
alright !